app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -31,7 +31,8 @@ def import_vm(source_vm_id, target_params)
   end
 
   def validate_import_vm
-    api_version >= '4.0'
+    version = highest_supported_api_version
+    version >= '4.0' if version
```

Comment:
Is this related to the VM being disconnected before `disconnect_storage` is called?
Feels like if this is an issue it is different and should be its own PR